Lib/unittest/mock.py

```diff
@@ -814,7 +820,7 @@ def _call_matcher(self, _call):
             else:
                 name, args, kwargs = _call
             try:
-                return name, sig.bind(*args, **kwargs)
+                return call(name, sig.bind(*args, **kwargs))
```

Comment:
Below construction looks more correct to me.

```python
bound_call = sig.bind(*args, **kwargs)
_Call((name, bound_call.args, bound_call.kwargs)
```

Reply:
Thank you, this feels much better. I was kind of abusing the idea of what a `_Call` was before, even if it was working.